exchange/utils.go

```diff
@@ -165,7 +165,7 @@ func (rs *requestSplitter) cleanOpenRTBRequests(ctx context.Context,
 		coreBidder, isRequestAlias := resolveBidder(bidder, requestAliases)
 
 		// apply bidder-specific schains
-		sChainWriter.Write(&reqCopy, bidder)
+		sChainWriter.Write(reqWrapperCopy, bidder)
```

Comment:
Should we use `reqWrapperCopy` instead of reqCopy further down?

Reply:
If there was another `sChainWriter.Write()` call further down, then it will not compile if the signature doesn't match. Otherwise my understanding is that this PR is just for `sChainWriter.Write()`

Reply:
There is just one usage of this function (not counting unit tests). Code compiles and runs. It is safe to merge.